deploy/olm-catalog/ibm-block-csi-operator-community/1.7.0/manifests/ibm-block-csi-operator.v1.7.0.clusterserviceversion.yaml

```diff
@@ -119,7 +119,7 @@ metadata:
         }
       ]
 spec:
-  displayName: "Operator for IBM block storage CSI driver"
+  displayName: "IBM block storage CSI driver plugin"
```

Comment:
Should be "IBM block storage CSI driver operator"

Reply:
done